drivers/pn532/pn532.c

```diff
@@ -175,7 +167,10 @@ static int _write(const pn532_t *dev, uint8_t *buff, unsigned len)
     if (dev->mode == PN532_I2C) {
 #ifdef PN532_SUPPORT_I2C
         i2c_acquire(dev->conf->i2c);
-        ret = i2c_write_bytes(dev->conf->i2c, PN532_I2C_ADDRESS, buff, len);
+        ret = i2c_write_bytes(dev->conf->i2c, PN532_I2C_ADDRESS, buff, len, 0);
+        if (ret == 0) {
+            ret = len;
```

Comment:
looking at the SPI part this might also be `ret = (int)len;`, though my compiler didn't complain about assigned unsigned to signed - but might be different with other versions?!